README.md

```diff
@@ -57,7 +57,6 @@ See [LICENSE][12] for further details.
 [4]: <https://coveralls.io/r/MushroomObserver/mushroom-observer?branch=master>
 [5]: <https://codeclimate.com/github/MushroomObserver/mushroom-observer.png>
 [6]: <https://codeclimate.com/github/MushroomObserver/mushroom-observer>
-[9]: <https://github.com/mo-nathan>
 [10]: <https://github.com/pellaea>
 [11]: <https://github.com/JoeCohen>
```

Comment:
We should get rid of 10-11 also, as there are no calls to them.
I'll take care of it tomorrow afternoon, but if you'd like to merge it sooner, please feel free to do so.